sycl/test-e2e/AddressSanitizer/out-of-bounds/local/multiple_source.cpp

```diff
@@ -14,21 +14,23 @@ constexpr std::size_t group_size = 1;
 
 #ifdef USER_CODE_1
 
+class MyKernelA;
 void foo(sycl::queue &Q, int *data) {
   Q.submit([&](sycl::handler &cgh) {
     auto acc = sycl::local_accessor<int>(group_size, cgh);
-    cgh.parallel_for<class MyKernel>(
+    cgh.parallel_for<MyKernelA>(
```

Comment:
Hi, could you please explain why did you make this changes?

Reply:
Kernel name types have to be forward declarable at namespace scope, and prior to the changes the test ended up using the same type for two different kernels.